app/Http/Controllers/Api/AuthController.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use Attribute;
+use App\Models\User;
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Validator;
+
+class AuthController extends Controller
+{
+     use ApiResponse;
+
+
+    //registering
+     public function register(Request $request)
+    {
+        $validate = Validator::make($request->all(), [
+            'name' => 'required|string|max:250',
+            'email' => 'required|string|max:250|email',
+            'password' => 'required|string|min:8|confirmed'
+
+        ]);
+
+        if($validate->fails()){
+         return $this->errorResponse('Validation Error!', 403);
+
+        }
+
+        $user = User::create([
+            'name' => $request->name,
+            'email' => $request->email,
+            'password' => Hash::make($request->password),
+            'is_patient' => false
```

Comment:
should be:
 'is_patient' => true,
because we are creating a patient record in user table